6-Typescript/server/tests/drivers/AppDriver.ts

```diff
@@ -1,62 +1,60 @@
-import { ITodo } from "../../interfaces/ITodo";
+import { ITodo } from "../../interfaces/interfaces";
 
 export class AppDriver {
     _httpClient: any;
     _url?: string;
-    _userId: string | undefined
+    _userId: string | undefined;
     constructor(httpClient: any, url: string) {
-        this._httpClient = httpClient,
-        this._url = url
-        this._userId = undefined
+        (this._httpClient = httpClient), (this._url = url);
     }
 
-    getTodoById(id:string) {
-        let headers: object = this.setHeadersForCookie();
-        return this._httpClient.get(`${this._url}${id}`, headers)
-    } 
+    getTodoById(id: string) {
+        const headers = this.createHeadersForCookie();
```

Comment:
can be done in `beforeEach`? there's a duplication here for getting the `headers`